test/algorithm/mh_unittest.cpp

```diff
@@ -96,6 +97,22 @@ TEST_F(mh_fixture, sample_unif_normal_posterior_mean_stddev)
     EXPECT_NEAR(sample_average(storage_2), 1.868814361437099766, 0.1);
 }
 
+TEST_F(mh_fixture, sample_unif_normal_posterior_mean_stddev_samples) {
+    auto model = (
+        theta |= uniform(-1., 2.), 
+        y |= normal(theta, 1.0) // {0.1, 0.2, 0.3, 0.4, 0.5}
+    );
+
+    auto start = std::chrono::high_resolution_clock::now();
```

Comment:
oops this was from my temporary benchmarking. Do you mind removing this xD

Reply:
Fixed in the latest version.